File: G-Nation-Jee/tag-ejb/src/main/java/tag/ejb/domain/Produit.java
package tag.ejb.domain;

import java.io.Serializable;
import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;


@Entity
public class Produit implements Serializable {

	
	private Long idProduit;
	private String name;
	private String description;
	private Double price;
	private Integer quantite;
	private String image_link;
	private Date date;
    private User user;
	
	public Produit(){
	}
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	public Long getIdProduit() {
		return idProduit;
	}

	public void setIdProduit(Long idProduit) {
		this.idProduit = idProduit;
	}

	@ManyToOne
	@JoinColumn(name="Id")
	public User getUser() {
		return user;
	}

	public void setUser(User user) {
		this.user = user;
	}

	private static final long serialVersionUID = 1L;

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getDescription() {
		return description;
	}

	public void setDescription(String description) {
		this.description = description;
	}

	public Double getPrice() {
		return price;
	}

	
	
	public Integer getQuantite() {
		return quantite;
	}

	public void setPrice(Double price) {
		this.price = price;
	}

	public void setQuantite(Integer quantite) {
		this.quantite = quantite;
	}

	public String getImage_link() {
		return image_link;
	}

	public void setImage_link(String image_link) {
		this.image_link = image_link;
	}
	@Temporal(TemporalType.TIMESTAMP)
	public Date getDate() {
		return date;
	}

	public void setDate(Date date) {
		this.date = date;
	}

	


}

File: G-Nation-Asp/TAG_Domain/Entities/produit.cs
using System;
using System.Collections.Generic;

namespace TAG_Domain.Entities
{
    public partial class produit
    {
        public produit()
        {
            this.panniers = new List<pannier>();
        }

        public long idProduit { get; set; }
        public Nullable<System.DateTime> date { get; set; }
        public string description { get; set; }
        public string image_link { get; set; }
        public string name { get; set; }
        public Nullable<double> price { get; set; }
        public Nullable<int> quantite { get; set; }
        public string idUser { get; set; }
        public virtual ICollection<pannier> panniers { get; set; }
        public virtual User user { get; set; }
    }
}
